readme.md

```diff
@@ -176,6 +179,37 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Placebo Effect](https://en.wikipedia.org/wiki/Placebo)
```

Comment:
this is already in master. It shouldn't be a part of your patch. 

Solution: Rebase your branch onto the latest master to clear this up.